app/controllers/hq/partners_controller.rb

```diff
@@ -6,6 +6,19 @@ def index
     @partners= Partner.all
   end
 
+  def new
+    @partner = Partner.new
```

Comment:
you could just call build_partner here and because there is no params none will be added, then we have the Partner.new call just in one place.